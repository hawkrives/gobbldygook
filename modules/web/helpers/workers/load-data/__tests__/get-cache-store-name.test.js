/* eslint-env jest */
// @flow

import getCacheStoreName from '../get-cache-store-name'

test('getCacheStoreName runs', () => {
    expect(() => getCacheStoreName('courses')).not.toThrow()
})

test('getCacheStoreName handles courses', () => {
    expect(getCacheStoreName('courses')).toMatchSnapshot()
})

test('getCacheStoreName runs', () => {
    expect(getCacheStoreName('areas')).toMatchSnapshot()
})

test('getCacheStoreName throws an error on unexpected values', () => {
    // $FlowFixMe
    expect(() => getCacheStoreName('invalid')).toThrowErrorMatchingSnapshot()
})
