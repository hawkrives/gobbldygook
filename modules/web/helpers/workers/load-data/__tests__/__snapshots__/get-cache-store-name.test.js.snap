// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getCacheStoreName handles courses 1`] = `"courseCache"`;

exports[`getCacheStoreName runs 1`] = `"areaCache"`;

exports[`getCacheStoreName throws an error on unexpected values 1`] = `"getCacheStoreName(): \\"invalid\\" is not a valid store type"`;
